fem/fespace.cpp

```diff
@@ -1059,7 +1059,8 @@ void FiniteElementSpace::Construct()
    bdofs[0] = 0;
    for (i = 0; i < mesh->GetNE(); i++)
    {
-      nbdofs += fec->DofForGeometry(mesh->GetElementBaseGeometry(i));
+      int geom = mesh->GetElementBaseGeometry(i);
+      nbdofs += geom == Geometry::POINT ? 0 : fec->DofForGeometry(geom);
```

Comment:
Don't insist on it, but to me it is more clear to say:

```c++
if (geom != Geometry::POINT)
{
   nbdofs += fec->DofForGeometry(geom);
}
```

Reply:
I agree, that is clearer. I'll change it.